dev-client/src/screens/SiteSettingsScreen/SiteSettingsScreen.tsx

```diff
@@ -61,53 +55,17 @@ export const SiteSettingsScreen = ({siteId}: Props) => {
 
   const onDelete = useCallback(async () => {
     await dispatch(deleteSite(site));
-    navigation.pop();
+    navigation.navigate('BOTTOM_TABS');
   }, [dispatch, navigation, site]);
 
+  if (!site) {
+    return null;
```

Comment:
Question: Since I'll be making a thing in the backlog to follow up on this, could you confirm if this understanding is correct?
1. When this happens, what will we render? --> Nothing for the SiteSettingsScreen, but that's fine because we will then immediately navigate to 'BOTTOM_TABS'
2. When does this happen? --> Every time we delete a site, because for not-yet-understood reasons it tries to re-render SiteSettingsScreen before navigating to 'BOTTOM_TABS'

Reply:
Thats my understanding.